build.sbt

```diff
@@ -122,6 +122,9 @@ lazy val api = crossProject(JVMPlatform, JSPlatform)
     mimaSettings,
     name := "twirl-api",
     jsEnv := nodeJs,
+    // hack for GraalVM, see: https://github.com/scala-js/scala-js/issues/3673 
+    // and https://github.com/playframework/twirl/pull/339
+    testFrameworks := List(new TestFramework("org.scalatest.tools.Framework","org.scalatest.tools.ScalaTestFramework")),
```

Comment:
@raboof, this is the right settings. 

I'm only changing it on `twirl-api` as it only impacts the scala.js part.